README.md

```diff
@@ -10,7 +10,7 @@ You can include this package in your preferred base image to make that base imag
 
 ## Requirements
 The Ruby Runtime Interface Client package currently supports Ruby versions:
- - 2.5.x up to and including 2.7.x
+ - 2.7.x
```

Comment:
@briensea I see that you closed this PR but are you planning to add Ruby 3 as a supported version as well?